app/assets/stylesheets/welcome.scss

```diff
@@ -0,0 +1,93 @@
+html, body {
+  height: 100%;
+	background-color: #333333;
+	background-image: image-url('kepler-bg-sm.gif');
+	background-size: cover;
+	font-family: 'Roboto', Helvetica, Verdana, Arial, sans-serif;
+}
+
+a:link {
+  text-decoration:none;
+	color: #666666;
+}
+
+a:visited {
+  text-decoration:none;
+	color: #666666;
+}
+
+a:hover {
+  text-decoration:none;
+	color: #CCCCCC;
+}
+
+a:active {
+  text-decoration:none;
+	color: #666666;
+}
+
+h1, h2, h3, h4, h5, h6 {
+  font-weight: normal;
+	color: #CCCCCC;
+}
+
+h1 {
+  font-style:normal;
+  font-weight:100;
+  text-align:center;
+  font-size: 48px;
+  letter-spacing:5px;
+  padding: 0;
+}
+
+h2 {
+  font-style:normal;
+	font-weight:300;
```

Comment:
Not a big deal, any idea what's up with the weird indentation spacing that's showing up?